apps/cookbook/src/app/examples/page-example/advanced/page-advanced-example.component.ts

```diff
@@ -48,12 +48,12 @@ const config = {
   <!-- Fixed Content -->
   <kirby-fab-sheet *kirbyPageContent="{fixed: true}" horizontalAlignment="right">
     <kirby-icon name="write-message"></kirby-icon>
-    <kirby-action-sheet
+    <kirby-action-sheet-popout
```

Comment:
Changed the API to avoid breaking change. It is now possible to optionally instruct the action-sheet to hide the trigger button. FAB and page showcases was updated to reflect this